src/tests/test_sexual_behaviour.py

```diff
@@ -77,3 +79,21 @@ def test_behaviour_updates():
         pop.sexual_behaviour.update_sex_groups(pop.data)
     subsequent_groupings = pop.data["sex_behaviour"]
     assert(any(initial_groupings != subsequent_groupings))
+
+
+def test_initial_sex_behaviour_groups():
+    N = 10000
+    pop_data = Population(size=N, start_date=date(1989, 1, 1)).data
+    probs = sbd.init_sex_behaviour
+    for sex in SexType:
+        index_sex = selector(pop_data, sex=(operator.eq, sex))
+        n_sex = sum(index_sex)
+        Prob_sex = probs[sex].copy()
+        Prob_sex /= sum(Prob_sex)
+        for g in SexBehaviours[sex]:
+            index_group = selector(pop_data, sex_behaviour=(operator.eq, g))
+            p = Prob_sex[g]
+            sigma = np.sqrt(p*(1-p)*float(n_sex))
+            E = float(n_sex)*p
+            N_g = sum(index_sex & index_group)
+            assert(((E - sigma*3) <= N_g) and (N_g <= (E + sigma*5)))
```

Comment:
Scalar numbers have convenient double-sided comparison!
```suggestion
            assert E - sigma*3 <= N_g <= E + sigma*5
```

(Also, I'd avoid parentheses after `assert`. You can end up asserting a tuple accidentally, which will always be "truthy" even if it contains only `False` values)

Reply:
Separately, is there a reason the interval is not symmetric?

Reply:
There is no reason the interval is not symmetric, I think that was just a typo. (I initially set *5 but thought that this was a bit too lenient as a test, and went down to *3, but evidently something went awry!)